File: seed/csharp-sdk/server-sent-events-openapi/Snippets/Example26.cs
using SeedApi;

public partial class Examples
{
    public async Task Example26() {
        var client = new SeedApiClient(
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.StreamXFernStreamingUnionAsync(
            new StreamXFernStreamingUnionRequest(
                new UnionStreamMessageVariant {
                    Prompt = "prompt",
                    Message = "message",
                    StreamResponse = false
                }
            ) {
                StreamResponse = false,
            }
        );
    }

}

File: seed/swift-sdk/server-sent-events-openapi/Tests/Snippets/Example26.swift
import Foundation
import Api

enum Example26 {
    static func snippet() async throws {
        let client = ApiClient(baseURL: "https://api.fern.com")

        _ = try await client.streamXFernStreamingUnion(request: StreamXFernStreamingUnionRequest.message(
            UnionStreamMessageVariant(
                streamResponse: false,
                prompt: "prompt",
                message: "message"
            )
        ))
    }
}
